src/Components/TimelineSidebar/TimelineSidebar.Component.jsx

```diff
@@ -50,9 +50,7 @@ class TimelineSidebar extends Component {
       }
       return user.assignedRole === 'assignee';
     });
-    if (this.state.assignee !== assignee) {
-      this.setState({ assignee });
-    }
+    this.setState({ assignee });
```

Comment:
Did you need to get rid of this, `If` check ?
- It seems to cut out the  issue of Re-assigning the same already assigned person to an Incident.

Reply:
Your point is valid attending to it